config/checkstyle/tree_walker_module_directives.xml

```diff
@@ -121,12 +121,6 @@
     <property name="format" value="\bpeek\b"/>
     <property name="message" value="Usage of the prohibited peek method"/>
 </module>
-<module name="Regexp">
-    <property name="illegalPattern" value="true"/>
-    <property name="ignoreComments" value="true"/>
-    <property name="format" value="\bSneakyThrows\b"/>
-    <property name="message" value="String interpolation does not work in methods marked with lombok.SneakyThrows, use lombok.Lombok#sneakyThrow instead"/>
```

Comment:
Is it no longer true?

Reply:
It was likely never true in the first place, I've only ever observed that in `binding` module and thought that's the fault of `@SneakyThrows`, but the actual reason was/is sth different... still dunno what specifically